mapcompose/src/main/java/ovh/plrapps/mapcompose/utils/Point.kt

```diff
@@ -0,0 +1,3 @@
+package ovh.plrapps.mapcompose.utils
+
+data class Point(val x: Double, val y: Double)
```

Comment:
Ideally this would be a [value class](https://kotlinlang.org/docs/inline-classes.html), but unfortunately currently value classes can only have one property. Hopefully one day this will be fixed: https://youtrack.jetbrains.com/issue/KT-1179

I tried creating a value class anyway, by packing the two `Double`s into a single `Long`: https://pl.kotl.in/camTTpjz5
It works fine but the math overhead is probably not worth it.

To be honest I doubt the allocations of such a small object would be any issue.